commons/build.gradle

```diff
@@ -172,24 +174,32 @@ task buildAcaPyJavaClient(type: GenerateTask) {
 
 task buildIssuerUiClient(type: GenerateTask) {
 	generatorName = "typescript-angular"
-	def inputPath = "${projectDir}/contract/api-issuer.yaml".toString()
+	def inputPath = "${project.buildDir}/contract/api-issuer.yaml".toString()
 	def outputPath = "${project.buildDir}/generated/openapi/issuer-ui-client".toString()
 	inputSpec = inputPath
 	outputDir = outputPath
 }
 
 task buildVerifierClient(type: GenerateTask) {
 	generatorName = "typescript-angular"
-	def inputPath = "${projectDir}/contract/api-verifier.yaml".toString()
+	def inputPath = "${project.buildDir}/contract/api-verifier.yaml".toString()
 	def outputPath = "${project.buildDir}/generated/openapi/verifier-ui-client".toString()
 	inputSpec = inputPath
 	outputDir = outputPath
 }
 
 task buildConnectorUiClient(type: GenerateTask) {
 	generatorName = "typescript-angular"
-	def inputPath = "${projectDir}/contract/api-connector.yaml".toString()
+	def inputPath = "${project.buildDir}/contract/api-connector.yaml".toString()
 	def outputPath = "${project.buildDir}/generated/openapi/connector-ui-client".toString()
 	inputSpec = inputPath
 	outputDir = outputPath
 }
+
+task setVcmsVersionToContractTemplates(type: Copy) {
+	from './contract'
+	include '*.yaml'
+	include '*.json'
```

Comment:
Do we really need json ?

Reply:
Yes, it is because of the ACA-Py OpenApi specification.The Webhook specification has references to the ACA-Py spec, as it handle the transition from aries data structure to our abstracted structure. As an alternative, we could also not version our internal OpenApi specs.